src/commonMain/kotlin/com/outsidesource/oskitkmp/lib/Bits.kt

```diff
@@ -0,0 +1,55 @@
+package com.outsidesource.oskitkmp.lib
+
+object Bits {
+
+    inline fun <reified T : Number> getBit(number: T, position: Int): Boolean {
+        val num = number.toLong()
```

Comment:
We should not be converting everything to a long and then back. That is unnecessary and a performance penalty.